File: src/main/java/com/back/nbe9112team06/domain/meeting/dto/FinalizeRequest.java
package com.back.nbe9112team06.domain.meeting.dto;

import com.fasterxml.jackson.annotation.JsonFormat;
import jakarta.validation.constraints.NotNull;
import java.time.LocalDate;
import java.time.LocalTime;

public record FinalizeRequest(
        @NotNull(message = "날짜를 입력해주세요.") LocalDate date,
        @NotNull(message = "시간을 입력해주세요.") @JsonFormat(pattern = "HH:mm") LocalTime time
) {
}

File: src/main/kotlin/com/back/nbe9112team06/domain/meeting/dto/request/FinalizeRequest.kt
package com.back.nbe9112team06.domain.meeting.dto.request

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDate
import java.time.LocalTime

data class FinalizeRequest(
    val date: LocalDate,

    @JsonFormat(pattern = "HH:mm")
    val time: LocalTime
)
